service/grails-app/views/license/_license.gson

```diff
@@ -23,7 +23,7 @@ if ( controllerName == 'license' && actionName.toLowerCase() == 'show' ) {
 }
 
 // The majority of this template is inherited from the parent template. Just add the extensions.
-json g.render(license, [includes: ['type', 'orgs', 'amendments'], expand: ['type', 'orgs', 'amendments'], excludes: ['alternateNames']]) {
```

Comment:
I think the excludes needs to be there if you are providing it yourself. Otherwise it will be output in the json twice?

Reply:
I was a bit confused about the `excludes` because I had to remove it in mod-agreements _subscriptionAgreement.gson to provide an empty array for alternateNames. So I found it more consistent to remove it from licenses also.

>  if you are providing it yourself

I'm not sure what you mean with that. If I call the agreements endpoint I only get the alternateNames once:
`curl -SsL -XGET -H 'X-OKAPI-TENANT: diku' "http://localhost:8081/licenses/licenses/${lid}" | jq`
```
{
  "id": "28f5ae01-134e-4a8e-bfe8-8d06506ba79e",
  "endDateSemantics": {
    "id": "02d3182871bc89340171bc8aea7f000a"
  },
  "dateCreated": "2020-04-27T16:49:14Z",
  "links": [
    {
      "id": "a97aece7-e6de-4e0a-b01f-1e8cd019e78f"
    }
  ],
  "customProperties": {},
  "contacts": [],
  "tags": [
    {
      "id": 15,
      "normValue": "legacy",
      "value": "legacy"
    }
  ],
  "lastUpdated": "2020-04-28T12:45:46Z",
  "alternateNames": [
    {
      "id": "f1a6ac75-ef4e-4361-9d4e-eb18a0a8795e",
      "name": "B-name",
      "owner": {
        "id": "28f5ae01-134e-4a8e-bfe8-8d06506ba79e"
      }
    },
    {
      "id": "a374e554-1137-4212-a67f-1734d4dfcc80",
      "name": "alternative name",
      "owner": {
        "id": "28f5ae01-134e-4a8e-bfe8-8d06506ba79e"
      }
    }
  ],
  "amendments": [],
  "docs": [],
  "name": "Test License 001",
  "orgs": [
    {
      "id": "982f38e0-ac72-4704-ac65-3b4526e30b54",
      "org": {
        "id": "6d832f8c-e8f9-4761-b62c-00162f99fc13",
        "orgsUuid": "1234-5678-9102-3356",
        "name": "EBSCO",
        "orgsUuid_object": {
          "error": 403,
          "message": "Forbidden"
        }
      },
      "owner": {
        "id": "28f5ae01-134e-4a8e-bfe8-8d06506ba79e"
      },
      "role": {
        "id": "02d3182871bc89340171bc8aead20013",
        "value": "licensor",
        "label": "Licensor"
      }
    },
    {
      "id": "37422bea-8997-49a3-8590-772d786c5f3b",
      "org": {
        "id": "eb8870e3-6c04-4e4e-93c5-c58e09784409",
        "orgsUuid": "1234-5678-9102-3355",
        "name": "Some Content Provider",
        "orgsUuid_object": {
          "error": 403,
          "message": "Forbidden"
        }
      },
      "owner": {
        "id": "28f5ae01-134e-4a8e-bfe8-8d06506ba79e"
      },
      "role": {
        "id": "02d3182871bc89340171bc8aeadf0014",
        "value": "licensee",
        "label": "Licensee"
      }
    }
  ],
  "type": {
    "id": "02d3182871bc89340171bc8aea540005",
    "value": "local",
    "label": "Local"
  },
  "status": {
    "id": "02d3182871bc89340171bc8aeaac000f",
    "value": "active",
    "label": "Active"
  },
  "supplementaryDocs": [],
  "description": "This is a test licenses",
  "startDate": "2019-01-01",
  "endDate": "2020-01-01",
  "_links": {
    "linkedResources": {
      "href": "/licenses/licenseLinks?filter=owner.id%3d28f5ae01-134e-4a8e-bfe8-8d06506ba79e"
    }
  },
  "openEnded": false
}

Reply:
But if you think it's necessary you should probably just close this PR and throw it away